make/dev.mk

```diff
@@ -17,3 +17,13 @@ endif
 		--wait --wait-for-jobs
 	kubectl rollout restart deployment capi-runtime-extensions
 	kubectl rollout status deployment capi-runtime-extensions
+
+dev.update-webhook-image-on-kind:
+ifndef SKIP_BUILD
+	$(MAKE) release-snapshot
+endif
+	kind load docker-image --name $(KIND_CLUSTER_NAME) \
+		ko.local/capi-runtime-extensions:$$(gojq -r .version dist/metadata.json)
+	kubectl set image deployment capi-runtime-extensions webhook=ko.local/capi-runtime-extensions:$$(gojq -r .version dist/metadata.json)
```

Comment:
I cant see any difference between this and 
```
helm upgrade --install capi-runtime-extensions ./charts/capi-runtime-extensions \
		--set-string image.repository=ko.local/capi-runtime-extensions \
		--set-string image.tag=$$(gojq -r .version dist/metadata.json) \
		--wait --wait-for-jobs
``` in the `dev.run-on-kind` target

Reply:
helm upgrade will install all other CR from the helm chart, this new target only updates the Deployment image.